lib/helpers.js

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const base58 = require('base58-universal');
+
+exports.splitKeyId = function splitKeyId({id}) {
+  // format: <keystoreId>/<localId>
+  const idx = id.lastIndexOf('/');
+  const localId = id.substr(idx + 1);
+  return {
+    keystoreId: id.substring(0, idx),
+    // convert to `Buffer` for storage savings (`z<base58-encoded ID>`)
+    localId: Buffer.from(base58.decode(localId.slice(1)))
```

Comment:
Since you're thinking about this, I spent a few minutes looking at optimal indexing strategies for `BinData`

https://docs.mongodb.com/manual/core/document/#the-_id-field

I ran across other resources that said that fixed length falling into the listed sizes there would be good.  So to that end, I don't know if it's worth adding any padding if these IDs are not already fixed length when in binary form.